content/docs/concepts/security/index.md

```diff
@@ -10,197 +10,230 @@ aliases:
-  the workload.
+Find out more about Istio authorization in the [authorization guide](/docs/concepts/security/#authorization).
+The guide for auditing is coming soon!
 
-- With domain name constraint, you can ensure service account uniqueness.
+### Security configuration
 
-### Communication security
+Istio makes it simple to configure and enable security features using Istio's rich yet easy-to-use policy creation
+and centralized policy management.
 
-Istio tunnels service-to-service communication through the client side
-[Envoy](https://envoyproxy.github.io/envoy/) and the server side Envoy.
-Istio secures end-to-end communication via:
+Find out more about configuring security policies in the [authentication](/docs/concepts/security/#authentication)
+and [authorization](/docs/concepts/security/#authorization) sections.
 
-- **Local TCP** connections between the service and Envoy.
+### Using other security mechanisms
 
-- **Mutual TLS** connections between proxies.
+While we strongly recommend using Istio security,
+Istio is flexible enough to allow you to plug in your own authentication and authorization mechanisms via the Mixer component.
+You can find out how to use and configure plugins in [Mixer policies and telemetry](/docs/concepts/policies-and-telemetry/#adapters).
 
-- **Secure Naming**: during the handshake process, the client side Envoy checks
-  that the service account the server side certificate provided is allowed
-  to run the target service.
+### Upcoming features
 
-### Key management
+- Identity/CA pluggability to support bring-your-own-CA and bring-your-own-identity.
+  For example, Vault integration, Active Directory integration.
 
-Istio supports services running on Kubernetes pods, virtual machines, and
-bare-metal machines. We use different key provisioning mechanisms for each
-scenario.
+- Perimeter security policies for Ingress/Egress proxy.
 
-For services running on Kubernetes pods, the per-cluster Citadel, acting as
-*Certificate Authority*, automates the key and certificate management process.
-Citadel mainly performs four critical operations:
+- Advanced auditing to meet various compliance requirements.
 
-- **Generate** a [SPIFFE](https://spiffe.github.io/docs/svid) key and
-  certificate pair for each service account
+- Secure developer/operator access from CORP to production services.
 
-- **Distribute** a key and certificate pair to each pod according to the
-  service account
+- Binary authorization integration to ensure the service is running with the trusted binary.
 
-- **Rotate** keys and certificates periodically
+## Istio identity
 
-- **Revoke** a specific key and certificate pair when necessary
+Identity is a fundamental concept of any security infrastructure. At the beginning of a service-to-service communication,
+the two parties need to exchange credentials consisting of their identity information, for mutual authentication purposes.
```

Comment:
Simplify and fix grammar:
Identity is a fundamental concept of any security infrastructure. At the beginning of a service-to-service communication, the two parties must exchange credentials with their identity information for mutual authentication purposes.